bloom/generators/debian/templates/control.em

```diff
@@ -12,3 +12,10 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
+
+Package: @(Package)-dbg
+Architecture: any
+Section: debug
+Priority: extra
+Depends: @(Package) (= ${binary:Version}), ${shlibs:Depends}, ${misc:Depends}
```

Comment:
@j-rivero, thanks for the feedback! You're absolutely right, shlibs isn't needed for the dbg package. It must have crept in out of habit. I'll push an update shortly to remove it.